chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -187,6 +177,26 @@ impl Actor for PeerManagerActor {
 
         let state = self.state.clone();
         let clock = self.clock.clone();
+        ctx.spawn(wrap_future(async move {
+            let mut interval =
+                tokio::time::interval(UPDATE_ROUTING_TABLE_INTERVAL.try_into().unwrap());
+            interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
+            loop {
+                let _timer = metrics::PEER_MANAGER_TRIGGER_TIME
+                    .with_label_values(&["update_routing_table"])
+                    .start_timer();
+                interval.tick().await;
```

Comment:
Hm, do we actually want to include `.tick()` into the timer's span?

Reply:
right, fixed.